.storybook/manager.js

```diff
@@ -6,4 +6,7 @@ import {createStorybookTheme} from './storybook-manager-theme';
 addons.setConfig({
     panelPosition: 'right',
     theme: createStorybookTheme('Movistar'),
+    toolbar: {
+        title: {hidden: true},
```

Comment:
This hides this element
![Screenshot 2023-10-26 at 18 09 52](https://github.com/Telefonica/mistica-web/assets/6722153/7bd54106-c5e6-40c2-b0a2-5c24bd662c53)